python/lsst/sconsUtils/dependencies.py

```diff
@@ -334,6 +348,72 @@ def configure(self, conf, packages, check=False, build=True):
                     return False
         return True
 
+    @staticmethod
+    def _get_config_var(name):
+        """The relevant Python is not guaranteed to be the Python
+        that we are using to run SCons so we must shell out to the
+        PATH python."""
+        pycmd = 'import distutils.sysconfig as s; print(s.get_config_var("{}"))'.format(name)
+        result = subprocess.check_output(["python", "-c", pycmd]).decode().strip()
+        # Be consistent with native interface
+        if result == "None":
+            result = None
+        return result
+
+    def configurePython(self, conf, packages, check=False, build=True):
+        state.log.info("Configuring package '%s'." % self.name)
+        python3rdinclude = self._get_config_var("CONFINCLUDEPY")
```

Comment:
Is it worth _get_config_var caching the results so that if you ask for CONFINCLUDEPY twice it only has to fork python once?

Reply:
configurePython seems to have been extra code that is not needed because it exists in configs/python.cfg.

That, along with the extra conda logic in depedencies.py, has superseded the need for configurePython, so it's been removed.